packages/opentelemetry-metrics/src/ValueRecorderMetric.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as api from '@opentelemetry/api';
+import { Resource } from '@opentelemetry/resources';
+import { BoundValueRecorder } from './BoundInstrument';
+import { Batcher } from './export/Batcher';
+import { MetricKind } from './export/types';
+import { Metric } from './Metric';
+
+/** This is a SDK implementation of Value Recorder Metric. */
+export class ValueRecorderMetric extends Metric<BoundValueRecorder>
+  implements api.ValueRecorder {
+  protected readonly _absolute: boolean;
+
+  constructor(
+    name: string,
+    options: api.MetricOptions,
+    private readonly _batcher: Batcher,
+    resource: Resource
+  ) {
+    super(name, options, MetricKind.VALUE_RECORDER, resource);
+
+    this._absolute = options.absolute !== undefined ? options.absolute : true; // Absolute default is true
```

Comment:
It looks like the `ValueRecorder` is a non-absolute (based on my understanding that means it can record negative numbers, right?) instrument. https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/metrics/api.md#valuerecorder

Also, I believe these kinds of options are now encoded into the instruments themselves (e.g. UpDownCounter), and shouldn't be able to be set by a user.

Neither of these seem like blockers to me, as they aren't blockers but should spawn a couple issues if not addressed in this PR.

Reply:
But this was extracted from a bigger file and was split into separate classes. So if anything needs to be changed can be done in different PR, not here. I have splitted the file into single classes, didn't modify them

Reply:
Sounds good! I made issues to track these: #1240 and #1241

This can be marked as resolved.